packages/design-system/src/components/label.tsx

```diff
@@ -7,10 +7,16 @@ import type { ComponentProps, ReactNode, Ref } from "react";
 import { forwardRef } from "react";
 import { textVariants } from "./text";
 import { styled, theme } from "../stitches.config";
+import { Label as RadixLabel } from "@radix-ui/react-label";
 
 export const labelColors = ["default", "preset", "local", "remote"] as const;
 
-const StyledLabel = styled("label", {
+const StyledLabel = styled(RadixLabel, {
```

Comment:
Do we really need to use radix for this? It kinda does nothing.

Reply:
dunno how valuable, it does one thing

https://github.com/radix-ui/primitives/blob/main/packages/react/label/src/Label.tsx#L24

Reply:
not one, it supports as button, it sets role label